### DISCUSSION TITLE
Error ProjectVic

### DISCUSSION THREAD

**Discussion by fewirchen** [sepinf-inc/IPED#944] [Q&A]
Please, I’ve downloaded a set of hashes from Project Vic (2020-02-08 14-52 to 2022-01-06 09-37.json), I've set enableProjectVicHashLookup and projectVicHashSetPath but I'm receiving the error "Processing Error: Unknown ProjectVic json data model!".

Is the IPED 3.18.12, thanks.

**Comment by lfcnassif** [comment]
Could you please share the beginning of your ProjectVIC json? Specifically the "odata.metadata" field value in the json. The feature was tested with http://github.com/ICMEC/ProjectVic/DataModels/1.3.xml#Media datamodel version. Possibly the datamodel was updated and we need to test with the last model and maybe update the code to handle it.

**Comment by fewirchen** [comment]
As requested, 

{
"@odata.context":"http://github.com/VICSDATAMODEL/ProjectVic/DataModels/2.0.xml/Default/$metadata#Media","value":[
    {
      "MediaID": 1,
      "Category": 0,
      "MD5": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
      "SHA1": "yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy",
      "MediaSize": 7763,
      "DateUpdated": "2022-01-06T13:37:27.9586572+00:00",
      "IsPrecategorized": false
    },
...

**Comment by wladimirleite** [comment]
@lfcnassif, I can take a look at this, as it probably also affects the new way of dealing with hashes (HashDB) implemented in IPED 4.

First I need to request the access to download the JSON, not sure how long this process takes. 
Let me know if you already have this new version.

**Reply by lfcnassif** [reply]
Thanks @tc-wleite, if you have time to look at it, great! If not, let me know, I could take a look after finishing other works, #937 is giving me some headaches...

I will download the last ProjectVIC json and, if you have time, share with you.

**Comment by lfcnassif** [comment]
I've just downloaded the lastest ProjectVIC diff file, and it came with "odata.metadata":"http://github.com/ICMEC/ProjectVic/DataModels/1.3.xml#Media

Looking at Hubstream Intelligence Agent software settings, you can choose the datamodel version, 2.0 is a choice, but the default is 1.3 to me:
![image](https://user-images.githubusercontent.com/7276994/151420032-18da289e-3bb9-40f9-9507-ced113afc448.png)

I don't know if the default changed in recent versions of the download software (mine is 3.7.6.0). @fewirchen what is your Agent version? What is your default VIC model version? Could you try to configure and download the 1.3 datamodel version?

**Comment by fewirchen** [comment]
You are right, I switched to 1.3 (despite the 2.0 was the default for me) and the error vanished.

The version of Hubstream Agent is 3.7.6.0 too.

This ticket can be closed, thank you.

**Reply by wladimirleite** [reply]
Maybe allowing both data models, if it is simple enough, would be a good idea.
